README.md

```diff
@@ -48,25 +47,30 @@ To test the web-based UI:
 ```
 make test
 ```
-## Other installation and deployment options
 
-### Docker
+## Using the edgex-ui
+
+With a modern browser, navigate to http://yourdomain:4000 (change *yourdomain* to your server's hostname or ip). 
+> NOTE: the GUI is running in non-secure mode when the environment variable EDGEX_SECURITY_SECRET_STORE=false, if EDGEX_SECURITY_SECRET_STORE=true, the accesss token for edgex is required, see the edgex document how to get the token.
+> NOTE: make sure that EdgeXFoundry is already running, Please refer to [Edgex Services Get Started](https://github.com/edgexfoundry/edgex-go/blob/main/README.md). 
+
+## User tutorial
+[tutorial](https://docs.edgexfoundry.org/2.3/getting-started/tools/Ch-GUI/). 
 
-To start all the EdgeX Go microservices in Docker, run the following command in the root of the EdgeX Go directory:
+## Other installation and deployment options
+
+### Build your own Docker Containers
 
 ```
-make docker run_docker
+make docker
 ```
 
-With a modern browser, navigate to http://yourdomain:4000 (change *yourdomain* to your server's hostname or ip). 
-> NOTE: make sure that EdgeXFoundry is already running, Please refer to [Edgex Services Get Started](https://github.com/edgexfoundry/edgex-go/blob/main/README.md). 
-
 ### Snap 
 
 EdgeX UI is also available as a snap package. For more details on the snap, including how to install it, please refer to [EdgeX UI Snap](./snap/README.md)
 
+
 ## Community
-- [EdgeXFoundry Chinese Club](https://www.edgexfoundry.club)
```

Comment:
FYI, there is an useful website: [EdgeXFoundry Chinese Website](https://cn.edgexfoundry.org/)